pkg/trace/propagation.go

```diff
@@ -127,6 +131,12 @@ func (t *otelTracer) fromHeaders(ctx context.Context, hdrs map[string][]string,
 	force := header.Get(HeaderForceTracing)
 	if force != "" {
 		ctx = ForceTracing(ctx)
+	} else {
+		// We do this in an else block so force-trace always takes precedent over force-no-trace.
+		forceNoTracing := header.Get(HeaderForceNoTracing)
+		if forceNoTracing != "" {
+			ctx = ForceNoTracing(ctx)
```

Comment:
This should just skip creating the `otelhttp.NewHandler` and just run `h.ServeHTTP(w, r)` here. Otherwise `otelhttp.NewHandler` will continue propagating the handler.

Reply:
If you don't want to completely drop the trace, then you can manually extract it like https://github.com/getoutreach/gobox/blob/main/pkg/trace/propagation.go#L140-L141. Just don't start a new span. 

I could see this header working either way. But to me it seems more intuitive to have this completely drop the trace is you pass a no tracing header.

Reply:
i think you thought this was in another function, and i added it in the appropriate function